// constants/index.js
export const FETCH_PHONES_PENDING = 'FETCH_PHONES_PENDING'
export const FETCH_PHONES_FULFILLED = 'FETCH_PHONES_FULFILLED'
export const FETCH_PHONES_REJECTED = 'FETCH_PHONES_REJECTED'
export const LOAD_MORE_PHONES_PENDING = 'LOAD_MORE_PHONES_PENDING'
export const LOAD_MORE_PHONES_FULFILLED = 'LOAD_MORE_PHONES_FULFILLED'
export const LOAD_MORE_PHONES_REJECTED = 'LOAD_MORE_PHONES_REJECTED'
export const FETCH_PHONE_BY_ID_PENDING = 'FETCH_PHONE_BY_ID_PENDING'
export const FETCH_PHONE_BY_ID_FULFILLED = 'FETCH_PHONE_BY_ID_FULFILLED'
export const FETCH_PHONE_BY_ID_REJECTED = 'FETCH_PHONE_BY_ID_REJECTED'
export const ADD_PHONE_TO_BASKET = 'ADD_PHONE_TO_BASKET'
export const SEARCH_PHONE = 'SEARCH_PHONE'
export const FETCH_CATEGORIES_PENDING = 'FETCH_CATEGORIES_PENDING'
export const FETCH_CATEGORIES_FULFILLED = 'FETCH_CATEGORIES_FULFILLED'
export const FETCH_CATEGORIES_REJECTED = 'FETCH_CATEGORIES_REJECTED'
export const REMOVE_PHONE_FROM_BASKET = 'REMOVE_PHONE_FROM_BASKET'
export const CLEAN_BASKET = 'CLEAN_BASKET'
